install-devenv.sh

```diff
@@ -25,6 +30,7 @@ esac
 
 archive="cpython-3.11.4+20230726-${platform}-apple-darwin-install_only.tar.gz"
 
+# Question: What make a temp directory and then rm -rf it?
```

Comment:
Don't quite understand this question? You've described what happens in the following 2 lines.

Reply:
it's used as a temporary place to download the archive to, then checksum check it, before finally unpacking it into the desired place

now that i'm thinking about it more, it would be better to download to `~/.cache/sentry-dev` actually